Podfile

```diff
@@ -10,5 +10,6 @@ target 'CourseGrab' do
   pod 'SnapKit'
   pod 'SPPermissions/Notification'
   pod 'Tactile'
+  pod 'Wormholy'
```

Comment:
we should change this to `pod 'Wormholy', :configurations => ['Debug']` so that when we archive and release the app, users can't see our network requests